jekyll/_cci2/about-circleci.md

```diff
@@ -11,74 +10,83 @@ version:
 - Server v2.x
 ---
 
-This document provides a summary of continuous integration and how CircleCI enables engineering teams with automation. CircleCI automates your software builds, tests, and deployments.
+Our mission is to empower technology-driven organizations to do their best work. We want to make engineering teams more productive with software builds, tests, and deployments through intelligent automation.
```

Comment:
I find this sentence a bit hard to scan but am struggling with an alternative: "We want to make engineering teams more productive with software builds, tests, and deployments through intelligent automation." ... maybe `We make engineering teams more productive. Build, test, and deploy using intelligent automation.` ... or something?